javascript/ql/src/Security/CWE-834/examples/TaintedLength.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var express = require('express');
+var router = new express.Router();
+var rootRoute = router.route('foobar');
+
+rootRoute.post(function(req, res) {
+    var obj = req.body;
+
+    calcStuff(obj);
```

Comment:
Since these examples will be embedded in the help page, please try to shorten them a bit:
- Remove the `'use strict'` directive.
- Inline `calcStuff`
- Use `var app = express();` and `app.post("/foo", (req, res) => ...)` instead of using routers.